authority-portal-backend/authority-portal-quarkus/src/main/kotlin/de/sovity/authorityportal/broker/dao/pages/catalog/models/DataOfferListEntryRs.kt

```diff
@@ -19,6 +19,7 @@ import java.time.OffsetDateTime
 data class DataOfferListEntryRs(
     val assetId: String,
     val assetTitle: String,
+    val assetUiJson: String,
```

Comment:
As said above, you can just return the `dataSourceAvailability` itself instead of the whole asset